Spawn-App-iOS-SwiftUI/Views/ProfileView.swift

```diff
@@ -13,55 +13,147 @@ struct ProfileView: View {
+                       ProfileField(label: "Username", value: appUser.username)
+                       ProfileField(label: "Email", value: appUser.email)
+                       BioField(label: "Bio", bio: Binding(
+                           get: { appUser.bio ?? "" },
+                           set: { appUser.bio = $0 }
+                       ))
+                   }
+                   .padding(.horizontal)
                     
-                    Text(NameFormatterService.shared.formatName(appUser: appUser))
-                        .font(.headline)
+                    Spacer()
+                    Divider().background(universalAccentColor)
+                    Spacer()
                     
-                    
-                    // Bio
-                    if let bio = appUser.bio {
-                        Text("Bio: \(bio)")
-                            .font(.body)
+                    // Edit Button
+                    Button(action: {
+                        // Edit button action
+                    }) {
+                        Text("Edit")
+                            .font(.headline)
+                            .foregroundColor(universalAccentColor)
+                            .frame(maxWidth: 135)
+                            .padding()
+                            .background(
+                                RoundedRectangle(cornerRadius: 20)
+                                    .stroke(Color.black, lineWidth: 1)
+                            )
                     }
-                    
-                    // Friend Tags
-                    if let friendTags = appUser.friendTags, !friendTags.isEmpty {
-                        Text("Friend Tags:")
+                    .padding(.horizontal)
+
+                    Spacer()
+                    Spacer()
+                    Spacer()
+                    Spacer()
+
+                    // Logout Button
+                    Button(action: {
+                    }) {
+                        Text("Log Out")
                             .font(.headline)
-                        ForEach(friendTags) { tag in
-                            HStack {
-                                Circle()
-                                    .fill(tag.color)
-                                    .frame(width: 10, height: 10)
-                                Text(tag.displayName)
-                            }
-                        }
+                            .foregroundColor(.white)
+                            .padding()
+                            .frame(maxWidth: 170)
+                            .background(profilPicPlusButtonColor)
+                            .cornerRadius(20)
                     }
+                    .padding(.horizontal)
+                    
+                    
+                    // commented out friend tags and last location for now as it's not included in the Figma Design
```

Comment:
Could you please delete the code instead of commenting it? I won't be sad if you delete my code bro, don't worry lol

Reply:
Deleted it for you